packages/messages/src/index.js

```diff
@@ -1,5 +1,10 @@
+// Internal dependencies
+const { makeListUserChatsService } = require('./use-cases');
 const welcome = require('./utils/welcomPackage');
+const { ChatModelMock, MessageModelMock } = require('../__tests__/testUtils');
 
 module.exports = {
   messagesWelcome: welcome,
+  // TODO: Remove mocks when domains are ready
+  listUserChats: makeListUserChatsService(ChatModelMock, MessageModelMock),
```

Comment:
@JoseNoriegaa instead of using testing mocks, we should create an additional folder called `utils/fixtures` and use the implementation that you have defined for tests.